src/bin/cql-stress-cassandra-stress/settings/param/mod.rs

```diff
@@ -30,7 +30,7 @@ pub trait ParamImpl {
         &self,
         param_name: &'static str,
         description: &'static str,
-        default_value: Option<&'static str>,
+        default_value: &Option<String>,
```

Comment:
- Using `&Option<T>` rather than `Option<&T>` is more awkward to use; e.g. to `map` over a value of the first type one must always call `as_ref()` first.
- Why pass a reference to an owned type? `Option<&str>` is always more flexible than `Option<&String>` or `&Option<String>`. You can always convert a `&String` to `&str` but not the other way around.